prediction_market_agent/tools/mech/utils.py

```diff
@@ -79,6 +84,16 @@ def mech_request_local(question: str, mech_tool: MechTool) -> OutcomePrediction:
                 "google_engine_id": keys.google_search_engine_id.get_secret_value(),
             },
         )
+    elif mech_tool in [MechTool.PREDICTION_ONLINE_SME, MechTool.PREDICTION_OFFLINE_SME]:
+        response = prediction_request_sme.run(
```

Comment:
Can we write a test for `prediction_request_sme.run` and place it under `integration_tests/`?

Reply:
It's covered already by this test https://github.com/gnosis/prediction-market-agent/blob/main/tests/mech/test_mech_local.py